src/utils.ts

```diff
@@ -67,7 +67,9 @@ export const getCleanClone = (object: any, realm: FastCopy.Realm): any => {
   }
 
   if (~toStringFunction.call(object.constructor).indexOf('[native code]')) {
-    return new object.constructor();
+    try {
+      return new object.constructor();
```

Comment:
this is the real change, wrapping things in a `try` to allow graceful fallback to creating the object via `Object.create()`